setup.cfg

```
@@ -1,5 +1,42 @@
+[metadata]
+name = django-hijack
+author = arteria GmbH
+author_email = admin@arteria.ch
+description = django-hijack allows superusers to hijack (=login as) and work on behalf of another user.
+long_description = file: README.md
```

long_description_content_type is misssed
```suggestion
long_description = file: README.md
long_description_content_type = text/markdown
```